app/scripts/modules/amazon/src/pipeline/stages/deployCloudFormation/evaluateCloudFormationChangeSetExecutionMarkerIcon.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { IExecutionMarkerIconProps } from 'core/pipeline/config/stages/common/ExecutionMarkerIcon';
+
+export class EvaluateCloudFormationChangeSetExecutionMarkerIcon extends React.Component<IExecutionMarkerIconProps> {
+  constructor(props: IExecutionMarkerIconProps) {
+    super(props);
+  }
+
+  public render() {
+    if (
+      this.props.stage.isRunning &&
+      this.props.stage.stages[0].context.changeSetContainsReplacement &&
+      this.props.stage.stages[0].context.actionOnReplacement === 'ask'
+    ) {
+      require('./evaluateCloudFormationChangeSetExecutionDetailsAsk.less');
+      if (
+        typeof document.getElementsByClassName('execution-marker-running stage-type-deploycloudformation')[0] !==
```

Comment:
I am wondering if there is another was to do this. Is there some data from elsewhere in deck that would indicate if this were present? If so, we can pass that in as a prop.

Reply:
Not longer required https://github.com/spinnaker/deck/pull/7671/commits/b088c914c75463bf966663c866624f61473cb9fa